style.css

```diff
@@ -0,0 +1,23 @@
+* {
+  font-family: 'Open Sans', sans-serif;
+}
+
+.main-section {
+  border-top: 8px solid #F5C85F;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  background-color: white;
+  padding: 40px;
+  flex-wrap: wrap;
+}
+
+.vertical-line {
+  display: flex;
+  margin: 10px;
```

Comment:
you have this property repeated in line 22

Reply:
:shame: